composer.json

```
@@ -67,7 +66,10 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.7-dev"
+            "dev-master": "2.0-dev"
+        },
+        "symfony": {
+            "require": "^6.4 || ^7.1"
```

```suggestion
            "require": "^6.4"
```
It's either or, with 2 configured at once weird things tend to happen